common/geo/geoUtils.js

```diff
@@ -0,0 +1,79 @@
+const R = require('ramda')
+/**
+ * Projected coordinate reference systems
+ * format: {wkid: id, name:"CRS name", wkt: "Well Know Text"}
+ */
+const projected = require('@esri/proj-codes/pe_list_projcs.json')
+/**
+ * Geographic coordinate reference systems
+ * format: same as projected
+ */
+const geographic = require('@esri/proj-codes/pe_list_geogcs.json')
+const proj4 = require('proj4')
+const isValidCoordinates = require('is-valid-coordinates')
+
+const Srs = require('./srs')
+const ObjectUtils = require('../objectUtils')
+const NumberUtils = require('../numberUtils')
+const StringUtils = require('../stringUtils')
+
+const invalidLonLatCoordinates = [0, 90]
+
+const formatName = (name = '') => R.replace(/_/g, ' ')(name)
+
+/**
+ * Array of all srs.
+ * Every item has this format: {code: epsgCode, name: "Formatted coordinate reference system name"}
+ */
+const srs = R.pipe(
+  R.concat(projected.ProjectedCoordinateSystems),
+  R.concat(geographic.GeographicCoordinateSystems),
+  R.map(item => Srs.newSrs(item.wkid.toString(), formatName(item.name), item.wkt)),
+  R.sortBy(R.prop(Srs.keys.name))
+)([])
+
+const srsByCode = ObjectUtils.toIndexedObj(srs, Srs.keys.code)
+
```

Comment:
should all srs related code go to srs.js ?

Reply:
what I mean is that srs (maybe rename to srsArray ?), srsByCode, getSrsByCode should be moved to srs.js ?

Reply:
srs.js is used by the client too... I'll rename the internal constant into srsArray anyway

Reply:
can you write a comment then?

Reply:
ok